images/ironic-python-agent-rhel-9.yaml

```diff
@@ -0,0 +1,19 @@
+- imagename: ironic-python-agent
+  types: [qcow2]
+  elements:
+  - rhel
+  - selinux-permissive
+  - interface-names
+  - ironic-python-agent-ramdisk
+  - ironic-agent-multipath
+  - dynamic-login
+  - element-manifest
+  - enable-packages-install
+  - override-pip-and-virtualenv
+  - extra-hardware
+  - repo-setup
```

Comment:
Could you reorder these and add the comments to align with ironic-python-agent-centos-9-stream.yaml?